test/levels.test.js

```diff
@@ -550,3 +552,9 @@ test('changing level from info to silent and back to info in child logger', asyn
   result = await once(stream, 'data')
   check(is, result, expected.level, expected.msg)
 })
+
+test('trying to get levels when `this` is no longer a Pino instance returns an empty string', async ({ is }) => {
```

Comment:
this test is out of line with the others - and because of what it's testing (not Pino) needs to access the `getLevel` function directly on an anonymous object.

Reply:
why is this out of line with the others? Add a reference to the issue